File: backend/src/main/java/org/jdk/project/dto/platform/PlatformPageDto.java
package org.jdk.project.dto.platform;

import java.util.List;
import java.util.Map;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PlatformPageDto {
  private String title;
  private String description;
  private List<Map<String, Object>> columns;
  private List<Map<String, Object>> searchFields;
  private List<Map<String, Object>> formFields;
  private Map<String, Object> defaults;
}

File: backend/src/main/kotlin/org/jdk/project/dto/platform/PlatformPageDto.kt
package org.jdk.project.dto.platform

data class PlatformPageDto(
    var title: String? = null,
    var description: String? = null,
    var columns: List<Map<String, Any?>>? = null,
    var searchFields: List<Map<String, Any?>>? = null,
    var formFields: List<Map<String, Any?>>? = null,
    var defaults: Map<String, Any?>? = null,
)
